common-rest/src/main/java/org/uniprot/api/common/repository/store/TupleStreamTemplate.java

```diff
@@ -34,12 +40,17 @@
 public class TupleStreamTemplate {
     private final StreamerConfigProperties streamConfig;
     private final HttpClient httpClient;
+    private final SolrClient solrClient;
+    private final SolrCollection collection;
```

Comment:
we already have collection in the streamConfig attribute that we have in line 41

Reply:
woops thanks!